css/class-naming-conventions/readme.md

```diff
@@ -123,21 +123,80 @@ Sometimes you'll write a modifier for a component and you want that modifier to
 </div>
 ```
 
+
+#### Nested inside the component
```

Comment:
Also since we are mentioning there are a few different methods, would it be helpful to include a little blurb on which types of situation one method might be preferable over another? I know we did talk about that a bit in our meeting.